core/src/main/java/org/apache/calcite/sql/fun/SqlRollupOperator.java

```diff
@@ -79,14 +81,14 @@ private static void unparseCube(SqlWriter writer, SqlCall call) {
             writer.startList(SqlWriter.FrameTypeEnum.SIMPLE, "(", ")");
         for (SqlNode operand2 : ((SqlCall) operand).getOperandList()) {
           writer.sep(",");
-          operand2.unparse(writer, 0, 0);
+          operand2.unparse(writer, PRECEDENCE + 1, PRECEDENCE + 1);
```

Comment:
can you use `getLeftPrec()` and `getRightPrec()` (inherited from `SqlOperator`)? then you can remove `PRECEDENCE`.